.travis.yml

```diff
@@ -18,4 +18,5 @@ script:
 
 after_success:
   - ./scripts/publish_docs.sh
+  - ./scripts/cleanup.sh
```

Comment:
Cleanup is not needed as travis ci automatically cleans up the working directory.